### PR TITLE
[MeshingApplication] Fix tests after new Model

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/MeshingApplication/tests/test_remesh_sphere.py [hunk 1]
```diff
@@ -269,6 +270,7 @@ def test_remesh_sphere_skin(self):
         }
         """)
 
+        check_parameters["input_file_name"].SetString(file_path + "/" + check_parameters["input_file_name"].GetString())
```
Comment (philbucher) [inline applications/MeshingApplication/tests/test_remesh_sphere.py]: please use os.path.join, such things fail in windows

**Review by philbucher — CHANGES_REQUESTED** [review]
minor

File: applications/MeshingApplication/tests/test_remesh_sphere.py (hunk 1 above)
Comment (loumalouomega) [reply applications/MeshingApplication/tests/test_remesh_sphere.py]: You can suggest a change with the new GitHub interface, we can try

Comment (loumalouomega) [reply applications/MeshingApplication/tests/test_remesh_sphere.py]:
```suggestion
        check_parameters["input_file_name"].SetString(file_path + "/" + check_parameters["input_file_name"].GetString())
```

Comment (philbucher) [reply applications/MeshingApplication/tests/test_remesh_sphere.py]: I saw that but was too lazy to try
Some other time :)

Comment (loumalouomega) [reply applications/MeshingApplication/tests/test_remesh_sphere.py]:
```suggestion
        check_parameters["input_file_name"].SetString(os.path.join(file_path, check_parameters["input_file_name"].GetString()))
```

**Comment by loumalouomega** [comment]
Done

**Review by philbucher — APPROVED** [review]

**Comment by loumalouomega** [comment]
Thanks

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**